src/UI/SignTransactionsModals/index.tsx

```diff
@@ -105,6 +105,6 @@ function SignTransactionsModals({
   }
 
   return null;
-}
+};
 
 export default SignTransactionsModals;
```

Comment:
I see that a lot of components still keep the default export. This is still needed ?